simulation/traffic_simulator/include/traffic_simulator/math/hermite_curve.hpp

```diff
@@ -57,11 +57,10 @@ class HermiteCurve
   double getLength(size_t num_points) const;
   double getLength() const { return length_; }
   boost::optional<double> getSValue(
-    geometry_msgs::msg::Point position, double threshold_distance = 1.0,
-    unsigned int initial_resolution = 30, unsigned int max_iteration = 30, double tolerance = 0.001,
-    bool autoscale = false) const;
+    const geometry_msgs::msg::Point & point, unsigned int initial_resolution = 30,
+    unsigned int max_iteration = 30, double tolerance = 0.03, bool autoscale = false) const;
```

Comment:
`unsigned int` => `std::size_t`

https://stackoverflow.com/questions/131803/unsigned-int-vs-size-t

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/commit/0b3461ec444280b0c20ed2b8e5b008d42e74cc5e